docs/Project-Configuration/Stepback-Bisection.md

```diff
@@ -0,0 +1,51 @@
+# Stepback Bisection
+Evergreen's stepback bisection performs stepback by continuously reducing the amount of commits needed to test by half. To enable it, go to the project settings 'general' tab and enable the 'Stepback Bisection' flag. Note that the 'Stepback' flag has to be enabled as well for bisection to take place.
+
+## Motivation
+Traditionally, Evergreen performed linear stepback only. This is tasking each previous commit one by one. If stepback bisection is enabled, Evergreen will use bisection instead of linear.
+Evergreen performs linear stepback by default, which takes O(n) steps in the worst case to find the offending commit. With bisection, Evergreen performs binary search on the remaining tasks, cutting down the worst time to O(logn) steps.
+
+## Definitions
+The following are all task-specific fields that populate separately for every task. Only tasks that are in the middle or have finished the bisection process will have these fields.
```

Comment:
The first task will have last failing and next stepback task though as well right?

Reply:
The first task will only have next stepback task (last failing would be itself).